ocfweb/stats/templates/stats/accounts.html

```diff
@@ -1,14 +1,94 @@
 {% extends 'base.html' %}
+{% load common %}
 {% load stats %}
 
 {% block content %}
     <div class="ocf-content-block">
         {% stats_navbar %}
-        <p>
-            <img class="img-responsive" src="{% url 'cumulative_accounts_graph' %}" />
-        </p>
-        <p>
-            <img class="img-responsive" src="{% url 'cumulative_group_accounts_graph' %}" />
-        </p>
+    <div id="total"></div>
+    <div id="group"></div>
```

Comment:
I think indenting these to match the other content in the `div` would make sense:

```
<div class="ocf-content-block">
    {% stats_navbar %}
    <div id="total"></div>
    <div id="group"></div>
</div>
```

Reply:
Fixed this, consolidated the highcharts options for the 2 graphs, and replaced time on the tooltips with years. I think it looks fine on mobile.